test/runtests.jl

```diff
@@ -1473,4 +1473,9 @@ if VERSION >= v"0.7-"
     @test !hasproperty(tlayout, :p)
 end
 
+if VERSION >= v"0.7.0"
```

Comment:
...but this does need a `@static` because the named tuples don't parse on 0.6.
```suggestion
@static if VERSION >= v"0.7.0"
```